spec/convection/control/stack_spec.rb

```diff
@@ -36,6 +36,28 @@ module Convection::Control
         azs.uniq!
         expect(azs.size).to be(1)
       end
+
+      it 'availability_zones can be ignored' do
+        subject.exclude_availability_zones = %w(eu-central-1a)
+        expect(subject.availability_zones).to contain_exactly('eu-central-1b')
```

Comment:
We don't actually call out to amazon to list zones. We stub the response so this should be fine based on the stub today. Even if they expand their zones our stub is hit not the API.